src/app/components/RelatedContentSection/index.tsx

```diff
@@ -68,8 +76,13 @@ const renderRelatedContentList = ({
     index,
   });
 
+  const headlineFirst = isHeadlineFirst(item);
+
   return (
-    <PromoItem css={styles.promoItem} key={ariaLabelledBy}>
+    <PromoItem
+      css={[headlineFirst ? styles.promoItemFullWidth : styles.promoItem]}
```

Comment:
Does this need to be inside an array - would this also work?
```suggestion
      css={headlineFirst ? styles.promoItemFullWidth : styles.promoItem}
```